client/src/components/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from "react";
+import dojoengine from "../assets/images/dojoengine.png";
+import starknet from "../assets/images/starknet.png";
+import dice1 from "../assets/images/dice-2.png";
+import dice2 from "../assets/images/dice-3.png";
+import "../styles/Footer.scss";
+
+const Footer = () => {
+  useEffect(() => {
+    const updateFooterPosition = () => {
+      const body = document.body;
+      const html = document.documentElement;
+      const footer = document.querySelector(".footer") as HTMLElement;
+
+      const documentHeight = Math.max(
+        body.scrollHeight,
+        body.offsetHeight,
+        html.clientHeight,
+        html.scrollHeight,
+        html.offsetHeight
+      );
+      const viewportHeight = window.innerHeight;
+
+      if (documentHeight <= viewportHeight) {
+        footer.style.position = "absolute";
+        footer.style.bottom = "0";
+      } else {
+        footer.style.position = "relative";
+      }
+    };
+
+    updateFooterPosition();
+    window.addEventListener("resize", updateFooterPosition);
+
+    return () => {
+      window.removeEventListener("resize", updateFooterPosition);
+    };
+  }, []);
+
+  return (
+    <footer className="footer">
+      <div className="footer__content">
+        <p>
+          Powered by Starknet{" "}
```

Comment:
I don't think "Starknet" needs to still be here since it is already on the image